### PR TITLE
Converts Player to IWorldSpace

### PR DESCRIPTION
**Pull request by xNexus-ACS**

### CODE REVIEW

**Comment by xNexus-ACS** [comment]
In testing process rn

File: Exiled.CustomRoles/API/Features/CustomRole.cs [hunk 1]
```diff
@@ -759,7 +759,7 @@ private void OnInternalDying(DyingEventArgs ev)
             if (Check(ev.Player))
             {
                 CustomRoles.Instance.StopRagdollPlayers.Add(ev.Player);
-                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, Quaternion.Euler(ev.Player.Rotation), ev.Player.DisplayNickname, NetworkTime.time));
+                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, Quaternion.Euler(ev.Player.Rotation.eulerAngles), ev.Player.DisplayNickname, NetworkTime.time));
```
Comment (Thundermaker300) [inline Exiled.CustomRoles/API/Features/CustomRole.cs:762]:
```suggestion
                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, ev.Player.Rotation, ev.Player.DisplayNickname, NetworkTime.time));
```
Should be able to simply use the rotation directly now, right?

**Comment by louis1706** [comment]
Don't merge because it's contain Breaking Change

**Comment by xNexus-ACS** [comment]
> Don't merge because it's contain Breaking Change

yeah, that's why I have it on draft

**Pull request closed without merging**